packages/turf-area/test.ts

```diff
@@ -21,6 +21,37 @@ const fixtures = fs.readdirSync(directories.in).map((filename) => {
   };
 });
 
+// fixtures
+
+const invalidPoly = geometry("Polygon", [
+  [
+    [0.0, 0.0],
+    [0.0, 0.0],
+    [0.0, 0.0],
+    [0.0, 0.0],
+    [0.0, 0.0],
+  ],
+]);
+
+// compiler does not pass coords less than 4 point long, so have to re-assign
+invalidPoly.coordinates = [
+  [
+    [101.0, 0.0],
+    [101.0, 0.5],
+    [101.5, 0.5],
+  ],
+];
```

Comment:
I would suggest creating an invalid Polygon geojson object directly if the helper is not allowing you to.

```
import { Polygon } from 'geojson'
const invalidPoly: Polygon = {
  type: "Polygon",
  coordinates: [
    [
      [101.0, 0.0],
      [101.0, 0.5],
      [101.5, 0.5],
    ],
  ]
};
```